code/src/index.css

```diff
@@ -1,13 +1,202 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  margin: 20px;
+  font-family: "Roboto Mono";
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
+html {
+  box-sizing: border-box;
+}
+
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: "Roboto Mono", monospace;
+}
+
+p {
+  font-size: 12px;
+}
+
+.thought-p {
+  font-weight: 600;
+  color: #333;
+}
+
+.form-title {
+  font-size: 11px;
+  font-weight: 600;
+  font-family: "Roboto Mono";
+  color: #333;
+}
+
+.h3 {
+  font-size: 12px;
+}
+
+/* send thought btn*/
+
+.send-btn {
+  margin-top: 10px;
+  border-radius: 50%;
+  width: 55%;
+  height: 40px;
+  display: block;
+  border-radius: 3em;
+  font-weight: 700;
+  color: rgb(63, 62, 62);
+  background-color: rgb(255, 173, 173);
+  font-family: "Roboto Mono";
+  text-align: center;
+  font-size: 10px;
+  margin-bottom: 10px;
+}
+
+/* red hearts in send thought btn*/
+
+.send-heart {
+  color: rgb(255, 64, 50);
+  margin-right: 5px;
+  margin-left: 5px;
+  font-size: 14px; /* emoji size is adjusted with font-size */
+}
+
+legend {
+  color: #333;
+  padding: 5px;
+  position: relative;
+  font-weight: 600;
+  color: rgb(255, 173, 173);
+  font-family: "Roboto Mono";
+  font-weight: 700;
+}
+
+fieldset {
+  padding: 10px 25px 20px 20px;
+  width: 320px;
+}
+
+fieldset > legend {
+  margin: -5px;
+}
+
+.textarea {
+  width: 95%;
+  height: 45px;
+  border: 1px solid rgb(207, 207, 207);
+  background-color: #fff;
+}
```

Comment:
If `resize:none;` is added to the textform it becomes static so you cant resize it by pulling it in the lower right corner :)